Structure_Engine/Create/SectionProperties/SteelSection.cs

```diff
@@ -41,12 +42,12 @@ public static partial class Create
         /***************************************************/
 
         [Description("Creates a steel I-section from input dimensions. Please note that all units are in S.I., that is meters [m].")]
-        [Input("height", "Full height of the section [m].")]
-        [Input("webThickness", "Thickness of the web [m].")]
-        [Input("flangeWidth", "Width of the top and bottom flange [m].")]
-        [Input("flangeThickness", "Thickness of the top and bottom flange [m].")]
-        [Input("rootRadius", "Optional fillet radius between inner face of flange and face of web [m].")]
-        [Input("toeRadius", "Optional fillet radius at the outer edge of the flange [m].")]
+        [Input("height", "Full height of the section.", typeof(Length))]
+        [Input("webThickness", "Thickness of the web.", typeof(Length))]
+        [Input("flangeWidth", "Width of the top and bottom flange.", typeof(Length))]
+        [Input("flangeThickness", "Thickness of the top and bottom flange.", typeof(Length))]
+        [Input("rootRadius", "Optional fillet radius between inner face of flange and face of web.", typeof(Length))]
+        [Input("toeRadius", "Optional fillet radius at the outer edge of the flange.", typeof(Length))]
         [Input("material", "Steel material to be used on the section. If null a default material will be extracted from the database.")]
         [Input("name", "Name of the steel section.")]
```

Comment:
All of these should include: This is required for most structural packages to create the section.